### PR TITLE
Updating Slide

### PR DESCRIPTION
**Pull request by IntellexLLC**
Release notes can be added here.
- [x] Intro to self
- [ ]Cool about 

> Blah  Blah
@selkins13 

Associated with issue #2

### CODE REVIEW

File: _slides/02-selkins13.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello World: My Name is Mike!
```
Comment (satishone) [inline _slides/02-selkins13.md:6]: It's nice to meet you, Can you also give us a cool fact about yourself?

File: _slides/02-selkins13.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
 ![orderedlistocat](https://octodex.github.com/images/orderedlistocat.png)
-{: .center}
+{: .Center}
```
Comment (EISStutz) [inline _slides/02-selkins13.md:9]: Checking out this change... Nice!

**Review by IntellexLLC — COMMENTED** [review]
Please approve at your earliest convenience.

File: _slides/02-selkins13.md (hunk 2 above)
Comment (EISStutz) [inline _slides/02-selkins13.md:9]: testing one to three four... Comment wants to go here <---

**Review by selkins13 — APPROVED** [review]

**Pull request merged**